
float64 positionX
float64 positionY
float64 positionZ
float64 orientationX
float64 orientationY
float64 orientationZ
float64 orientationW
---
bool possible